src/content/community/get-involved/open-research/index.md

```diff
@@ -0,0 +1,413 @@
+[Teku client](https://consensys.net/knowledge-base/ethereum-2/teku/)
+
+[Nimbus client](https://nimbus.team/)
+
+
+#### Recent research
+
+[Ethresear.ch Consensus](https://ethresear.ch/c/consensus/29)
+
+[Availability/Finality dilemma](https://arxiv.org/abs/2009.04987)
+
+[Single slot finality](https://ethresear.ch/t/a-model-for-cumulative-committee-based-finality/10259)
+
+[Proposer-builder separation](https://notes.ethereum.org/@vbuterin/pbs_censorship_resistance)
+
+
+### Execution clients
+
+The execution layer is concerned with executing transactions, running the EVM and generating execution payloads to pass to the consensus layer. There are many active areas of research, especially as execution clients are updated ready for the merge. This includes building out light client support, research into gas limits and incorporating new data structures (e.g. Verkle Tries). 
+
+#### Background reading
+
+[Nodes and Clients](https://ethereum.org/developers/docs/nodes-and-clients)
+
+#### Client implementations
+
+[Geth client](geth.ethereum.org)
+
+[Nethermind client](nethermind.io)
+
+[Besu client](https://consensys.net/quorum/developers/)
+
+[Erigon client](https://github.com/ledgerwatch/erigon)
+
+#### Recent research
+
+[Database optimizations](https://github.com/ledgerwatch/erigon/blob/devel/docs/programmers_guide/db_faq.md)
+
+[State expiry](https://notes.ethereum.org/@vbuterin/state_expiry_eip)
+
+[History management](https://eips.ethereum.org/EIPS/eip-4444)
+
+[Verkle Trees](https://vitalik.ca/general/2021/06/18/verkle.html)
+
+[Data availability sampling](https://github.com/ethereum/research/wiki/A-note-on-data-availability-and-erasure-coding)
+
+
+### Formal verification
+
+Formal verification is writing code to verify that Ethereum's consensus specifications are correct and bug-free. There is an executable version of the specification written in Python that requires maintenance and development. Further research can help to improve the Python implementation of the specification and add tools that can more robustly verify correctness and identify issues.
+
+#### Background reading
+
+[Introduction to formal verification](https://ptolemy.berkeley.edu/projects/embedded/research/vis/doc/VisUser/vis_user/node4.html)
+
+[Formal Verification (Intel)](https://www.cl.cam.ac.uk/~jrh13/papers/mark10.pdf)
+
+
+#### Recent research
+
+[Formal verification of the deposit contract](https://github.com/runtimeverification/deposit-contract-verification)
+
+[Formal verification of the Beacon Chain specification](https://github.com/runtimeverification/deposit-contract-verification)
+
+
+
+### Data science and analytics
+
+There is a need for more data analysis tools and dashboards that give detailed information about activity on Ethereum and the health of the network.
+
+### Background reading
+
+[Dune Analytics](https://dune.com/browse/dashboards)
+
+[Client diversity dashboard](https://clientdiversity.org/)
+
+#### Recent research
+
+
+
+## Apps and tooling
```

Comment:
I think Oracles could be a good topic for the application layer:
https://ethereum.org/en/developers/docs/oracles/

Reply:
ok added as a subheading